### ISSUE TITLE
How to pick sigma?

### ISSUE THREAD

**Issue by ex3ndr** [lucidrains/voicebox-pytorch#46]
Hey everyone, i am tryin to figure out what values of σ aka sigma is meant to be used during training? There are no mentioning of a specific value in papers for some reason.

**Comment by zvorinji** [comment]
The paper does mention 0.00001 as a starting point but unclear if that’s actually what they used. That said if testing out whether a different number could make the model converge faster, I’d go up (not down) and by orders of magnitude each test.

**Comment by ex3ndr** [comment]
Where did you get this number? I don't see it in the paper

**Comment by Subuday** [comment]
@ex3ndr 
It looks like he mixed up flow matching sigma with Learning Rate.
![image](https://github.com/lucidrains/voicebox-pytorch/assets/46067553/11b91af1-4af0-45b7-8b90-3a0f78d54cfb)

**Comment by Subuday** [comment]
@ex3ndr Flow Matching paper recommends to set it sufficiently small. Guys from MatchaTTS(which also uses Flow Matching) trained their model with 1e-5 value.
